src/main/java/io/nats/client/BaseConsumerContext.java

```diff
@@ -20,6 +20,11 @@
  * The Consumer Context provides a convenient interface around a defined JetStream Consumer
  */
 public interface BaseConsumerContext {
+    /**
+     * Gets the consumer name that was used to create the context.
+     * @return the consumer name
+     */
+    String getConsumerName();
```

Comment:
I decided to make the consumer name available for pull ordered consumers. This helps reporting, and underneath I will maintain the name on error. I originally didn't want the name exposed, but it's exposed on push ordered, and of course can be seen by management api/CLI. It's intended for visibility and reporting